src/core/crypto_key_mgmt.c

```diff
@@ -44,13 +44,17 @@ int32_t Crypto_Key_OTAR(void)
     int x = 0;
     int y;
     int32_t status = CRYPTO_LIB_SUCCESS;
-    int pdu_keys = (sdls_frame.pdu.pdu_len - 30) / (2 + KEY_SIZE);
+    // uint16_t pdu_len = (uint16_t) sdls_frame.pdu.hdr.pdu_len[1] << 8 | sdls_frame.pdu.hdr.pdu_len[0];
+    int pdu_keys = (sdls_frame.pdu.hdr.pdu_len - 30) / (2 + 32);
```

Comment:
It is safe to assume KEY_SIZE is static at 32 bytes now?

Reply:
To be referenced in new issue